website/src/css/custom.css

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Any CSS included here will be global. The classic template
+ * bundles Infima by default. Infima is a CSS framework designed to
+ * work well for content-centric websites.
+ */
+
+/* You can override the default Infima variables here. */
+:root {
+  --ifm-color-primary: #764ABC;
+  --ifm-color-primary-dark: #613D9A;
+  --ifm-color-primary-darker: #4C3078;
+  --ifm-color-primary-darkest: #362256;
+  --ifm-color-primary-light: #825AC2;
+  --ifm-color-primary-lighter: #9B7BCE;
+  --ifm-color-primary-lightest: #B49CDA;
+}
```

Comment:
I believe you can add some more css here to address the styling issue addressed by @markerikson 

> As a first quick glance: looking at the Advanced Tutorial, the code block font seems smaller, and the diff-language blocks are not getting highlighted.

The `--ifm-font-size-base` defaults to `15px` from Infima, the underlying CSS framework that Docusaurus 2 uses. Where the base font size in the current RSK site is `16px`.

Although, directly changing it to `16px` causes some of the navbar item to wrap text. So you may need to tweak there a little bit.

<img width="1901" alt="image" src="https://user-images.githubusercontent.com/2055384/67632531-6b1d7980-f8df-11e9-9dd7-9892f2af2a9f.png">

Also the default `--ifm-code-font-size` is `85%` which causes the code block to become smaller.

<img width="1878" alt="image" src="https://user-images.githubusercontent.com/2055384/67632537-96a06400-f8df-11e9-8602-d2d667ca2d90.png">

Meanwhile, the classes in the `diff` language blocks are properly picked up, but there's no styling. This also can be fixed by adding some custom styling -- just some quick thoughts, there can be plugins / themes that already pick this up.

<img width="1399" alt="image" src="https://user-images.githubusercontent.com/2055384/67632549-cfd8d400-f8df-11e9-934f-f72a6f130371.png">

Reply:
I believe we're currently using Monokai as the syntax highlighting theme.